api/src/main/java/org/apache/iceberg/FileFormat.java

```diff
@@ -71,9 +71,9 @@ public static FileFormat fromFileName(CharSequence filename) {
     return null;
   }
 
-  private static int lastIndexOf(char c, CharSequence seq) {
+  private static int lastIndexOf(char ch, CharSequence seq) {
```

Comment:
The style guide prevents the name of anything from being less than 2 characters long. So 1 character parameter names are disallowed.

Reply:
It's difficult to enforce that in some cases only, single-char names are allowed. I think we should enforce the standard globally, and the majority of times a single-char variable name is too opaque for readers.

Reply:
Though in specific method instances we can add `SuppressWarnings`.

Reply:
I don't mind using `ch` instead of `c` for cases, like this, but I'd like to use `SuppressWarnings` for cases where the variable name matches the spec, like `W` and `L` in transforms.

Reply:
Hm, so when I was editing the code to match the style guidelines, I found `W` and `L` to themselves be opaque, and I had to give a moment's thought to consider how to rename them since I had to derive what the semantic meaning was. Now, one can of course know what these variables mean in the context of the code they're in - it didn't take me long to understand what they were for. But I'd like for us to reconsider modifying the spec to make those variables fully descriptive of what they represent, so that they can be understood at a glance of either the spec or the code. Code should be self documenting as much as possible. Thoughts?

Reply:
If you didn't know what they meant, then let's give them better names. This is probably just a case where I'm familiar with the spec and I shouldn't assume other people are.